ibis/backends/duckdb/__init__.py

```diff
@@ -337,41 +337,29 @@ def get_schema(
         -------
         sch.Schema
             Ibis schema
-
         """
-        conditions = [sg.column("table_name").eq(sge.convert(table_name))]
-
-        if catalog is not None:
-            conditions.append(sg.column("table_catalog").eq(sge.convert(catalog)))
-
-        if database is not None:
-            conditions.append(sg.column("table_schema").eq(sge.convert(database)))
-
-        query = (
-            sg.select(
-                "column_name",
-                "data_type",
-                sg.column("is_nullable").eq(sge.convert("YES")).as_("nullable"),
+        query = sge.Describe(
```

Comment:
`DESCRIBE` in DuckDB used to be different than using `information_schema`, but only for CSV files, and that case has been fixed in a version that we no longer support.